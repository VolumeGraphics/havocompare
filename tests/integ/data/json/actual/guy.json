{ "name": "Takumi", "age": 18, "car": "Panda Trueno" }
