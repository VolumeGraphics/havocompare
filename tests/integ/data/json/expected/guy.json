{ "name": "Keisuke", "age": 21, "car": "RX7", "brothers": ["Ryosuke"] }
